src/Reflection/ParametersAcceptorSelector.php

```diff
@@ -515,6 +515,10 @@ private static function getCurlOptValueType(int $curlOpt): ?Type
 			return new UnionType([new ConstantIntegerType(0), new ConstantIntegerType(2)]);
 		}
 
+		if (defined('CURLOPT_POSTFIELDS') && $curlOpt === CURLOPT_SSL_VERIFYHOST) {
+			return new UnionType([new StringType(), new ArrayType(new MixedType(), new MixedType())]);
```

Comment:
Can the array key/value types be more precise then mixed?

Reply:
Honestly, I dunno.

Doc say
```
The full data to post in a HTTP "POST" operation. This parameter can either be passed as a urlencoded string like 'para1=val1&para2=val2&...' or as an array with the field name as key and field data as value. If value is an array, the Content-Type header will be set to multipart/form-data. Files can be sent using [CURLFile](https://www.php.net/manual/en/class.curlfile.php) or [CURLStringFile](https://www.php.net/manual/en/class.curlstringfile.php), in which case value must be an array.
```

So, value can be string, arrays, but maybe others things...

And keys, it's string. But I'm kinda worried bout the int keys.
```
$a[0] = 'foo';
$a[1] = 'bar';
```
which could be considered as '0=foo&1=bar'. (since '0' key would still be casted to int...)